ports/satlovecake/SATAN LOVES CAKE.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Set XDG_DATA_HOME if not already set
+XDG_DATA_HOME=${XDG_DATA_HOME:-$HOME/.local/share}
+
+# Check for PortMaster installation in various directories
+if [ -d "/opt/system/Tools/PortMaster/" ]; then
+  controlfolder="/opt/system/Tools/PortMaster"
+elif [ -d "/opt/tools/PortMaster/" ]; then
+  controlfolder="/opt/tools/PortMaster"
+elif [ -d "$XDG_DATA_HOME/PortMaster/" ]; then
+  controlfolder="$XDG_DATA_HOME/PortMaster"
+else
+  controlfolder="/roms/ports/PortMaster"
+fi
+
+# PortMaster info
+source $controlfolder/control.txt
+source $controlfolder/device_info.txt
```

Comment:
Mod files sourcing is missing and `export PORT_32BIT="Y"` must be set before.
Sourcing `device_info.txt` is not needed anymore.
```suggestion
export PORT_32BIT="Y"
[ -f "${controlfolder}/mod_${CFW_NAME}.txt" ] && source "${controlfolder}/mod_${CFW_NAME}.txt"
```